UI/js/Template.js

```diff
@@ -1,37 +1,37 @@
 function createdWhen(date) {
-  const when = new Date(Date.now()).getDay() - new Date(date).getDay();
-  if (when <= 10) {
-    return 'Today';
-  }
-  return `${when} days ago`;
+	const when = new Date(Date.now()).getDay() - new Date(date).getDay();
+	if (when <= 10) {
+		return 'Today';
+	}
+	return `${when} days ago`;
 }
 
 function happening(date) {
-  const monthNames = ['January', 'February', 'March', 'April', 'May', 'June',
-    'July', 'August', 'September', 'October', 'November', 'December',
-  ];
-  const d = new Date(date);
-  const day = d.getDay();
-  const month = monthNames[d.getMonth()];
-  const year = d.getFullYear();
-  const happens = `${day} / ${month} / ${year}`;
-  return `${happens}`;
+	const monthNames = ['January', 'February', 'March', 'April', 'May', 'June',
+		'July', 'August', 'September', 'October', 'November', 'December',
+	];
+	const d = new Date(date);
```

Comment:
How can you use `d` as a variable name?

Reply:
I used 'd' because i was referring to date. i apologize for wrong naming. i have implemented changes now. Thanks

Reply:
i used for loop (es5) because forEach wasnt working with async await as expected, forEach just fired multiple asynchorous calls without waiting for the previous call to resolve. I just implement the same function with map and it works....